scripts/Muon/GUI/MuonAnalysis/muon_analysis_2.py

```diff
@@ -62,6 +63,7 @@ def __init__(self, parent=None):
         except AttributeError as error:
             self.warning_popup(error.args[0])
 
+        # self.setAttribute(QtCore.Qt.WA_DeleteOnClose)
```

Comment:
Is this being kept for a reason?

Reply:
Good spot, this was left in by accident.